x-pack/test/fleet_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -526,6 +535,7 @@ const expectAssetsInstalled = ({
         { id: '47758dc2-979d-5fbe-a2bd-9eded68a5a43', type: 'epm-packages-assets' },
         { id: '318959c9-997b-5a14-b328-9fc7355b4b74', type: 'epm-packages-assets' },
         { id: 'e21b59b5-eb76-5ab0-bef2-1c8e379e6197', type: 'epm-packages-assets' },
+        { id: '4c758d70-ecf1-56b3-b704-6d8374841b34', type: 'epm-packages-assets' },
```

Comment:
hey @jgowdyelastic, I'm working on a similar PR https://github.com/elastic/kibana/pull/95885 for `security_rule` assets. how'd you generate these UUIDs? I looked through your commits and couldn't reverse engineer your process

Reply:
@rw-access  It took me quite a while to work this out, after adding lots of log statements in fleet I found the best way is to run the tests locally and then copy the IDs out of the errors.
They need to be in the correct order, which again I determined from the test errors.
I'm happy to help if this doesn't work for you.

You need to follow the steps here to run the tests:
https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/README.md#tests